### DISCUSSION TITLE
H1 - suddenly only fast forward and fast backward

### DISCUSSION THREAD

**Discussion by bestbean** [kachurovskiy/nanoels#110] [Q&A]
Since today, spindle is only turning if fast forward or fast backward is pressed.
Normal operation "on" with a value for carriage movement, e.g. 0.5 mm.

In fast forward or backward mode, stopps are respected.

**Comment by kachurovskiy** [comment]
Hi @bestbean, long time no see! 🙂 Sounds like your encoder wires have gone loose. If you change the display mode, do you see angle changing when you rotate the chuck by hand?

A video of the problem would help too :) Cheers!

**Comment by bestbean** [comment]
Good diagnosis!
encoder gear adapter was loose ....
